v3/newrelic/config.go

```diff
@@ -812,6 +817,15 @@ func (c Config) maxTxnEvents() int {
 	return configured
 }
 
+// maxSpan returns the configured maximum number of Span Events if it has been configured
+// and is less than the default maximum; otherwise it returns the default max.
+func maxSpanEvents(configured int) int {
```

Comment:
Should this be a method on the `Config` struct like the other `max*Events()` methods?